theia-full-docker/next.package.json

```diff
@@ -56,7 +56,8 @@
         "**/vscode-json-languageserver/**/vscode-languageserver": "6.0.0-next.1"
     },
     "devDependencies": {
-        "@theia/cli": "next"
+        "@theia/cli": "next",
+        "@theia/debug": "next"
```

Comment:
@chrbayer can you explain why `@theia/debug` was added as a `devDependency`? (it is already added as a dep)

Reply:
I have not noticed that "@theia/debug" is already in included in the dependencies, just copied it from the dart image. It will be removed :-)